keps/sig-windows/2578-windows-conformance/README.md

```diff
@@ -0,0 +1,474 @@
+- The way users run pods on Windows, as a specific user, varies from the implementation on Linux, as illustrated here: https://github.com/openshift/windows-machine-config-operator/pull/638. 
+
+We propose the following definitions as the Windows conformance standard for Kubernetes clusters that are released after Kubernetes 1.24 and onwards. This specification is to be updated as new features are added or removed from the Kubernetes API itself.
+
+### User Stories (Optional)
+
+#### Story 1
+
+As a Windows Application developer, I want to verify that the features I rely on in Kubernetes are supported on my specific Windows Kubernetes cluster.
+
+#### Story 2
+
+As a Kubernetes vendor, I want to evaluate the completeness of my Kubernetes support matrix in the context of Windows supported features in the Kubernetes API.
+
+#### Story 3
+
+As a Kubernetes developer, I want to immediately discern whether the features I want to add will affect the Windows support matrix. If there are any confirmed or potential impact(s) I want to be able to rapidly convey them in a canonical way to the broader community.
+
+#### Story 4
+
+As an IT administrator, I want to know whether my current version of Kubernetes supports ActiveDirectory or other Windows features.
+
+### Risks and Mitigations
+
+## Design Details
+
+### Specification of Windows Operational Readiness (Kubernetes 1.24)
+
+- In order to qualify as a Windows test, the pods that demonstrate functionality for a given attribute of a cluster must be running on a `pause` image. 
+
+Additionally, this image must be scheduled to an operating system that meets the following requirements:
+
+- Windows Build Versions:
+  - Windows Server 2019 (LTSC)
+  - Windows Server 2022 (LTSC)
+
+- Installed KBs:
+  - Kubernetes 1.21 and forward
+
+- All tests in the Core Windows Operational Readiness suite depend on `pause` as well as `agnhost`, and no other images.  This is because there are no requirements (thus far) in this specification which cannot be tested via the `pause` image, and thus it simplifies the types of pods we test on.  In the future we may changes this requirement if new workloads are important for operational readiness specification (i.e. a GPU intensive app or a CNI for host process verification).
+- All tests should use the `Pod.OS` specification field as the *only* required scheduling directive for pods, once this field is made GA.  Up until this time, the way tests schedule Windows pods can use RuntimeClasses, Labels, or taints in any way which "works" to implement the other requirements.
```

Comment:
pod.OS being used as scheduling is still up in the air afaik cc @ravisantoshgudimetla

Reply:
fixed, pushing shortly